spec/models/document_capture_session_spec.rb

```diff
@@ -2,7 +2,15 @@
 
 RSpec.describe DocumentCaptureSession do
   let(:doc_auth_response) do
-    DocAuth::Response.new(
+    Class.new(DocAuth::Response) do
```

Comment:
I don't see this syntax much in our other tests.  I am not sure how maintainable it is.  What's the benefit of this approach?

Reply:
@eileen-nava, was trying to mock some behavior, turns out not needed, removing it.